src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -97,14 +97,25 @@ const Comments = ({
       });
   };
 
+  const resolveComment = (commentId: string, resolved: boolean) => {
```

Comment:
Again, maybe rename to `setResolved` or `setCommentResolution` as this method can both resolve and unresolve comments